client/admin/App.jsx

```diff
@@ -15,37 +15,43 @@ const App = () => (
   </AdminContext>
 );
 
-const contentTypesSelector = createSelector(
-  [state => state.admin.resources['content-types']?.data || {}],
-  resources => Object.values(resources),
-);
-
 function Resources() {
   /**
    * Prefetch content-types to set dynamic resources
    */
   useQueryWithStore({
     type: 'getList',
     resource: 'content-types',
-    pagination: { page: 0 , perPage: 100 }
+    pagination: { page: 0, perPage: 100 },
   });
+
   const contentTypes = useSelector(contentTypesSelector);
+  const computedContentTypes = useRef({});
+  const [resources, setResources] = useState([]);
+  const contentTypesString = JSON.stringify(contentTypes);
+
+  useEffect(() => {
```

Comment:
again, why don't we revert to a previous solution with reselect to avoid infinite loop? it looks like it was simpler that current solution